File: sum_of_prime_digits.c
//C Program to Find the Sum of Prime Digits

#include<stdio.h>
int main(){
    int i,s=0,count,n,r;
    printf("Enter the number: ");
    scanf("%d",&n);
    while (n>0){
        r=n%10;
        if (n>1){
            count=0;
            for(i=2;i<=(r/2);i++){
                if ((r % i)==0){
                    count=count+1;
                    break;
                }
            }
            if (count==0){
                s=s+r;
            }
        }
        n=n/10;
    }
    printf("The Sum of the Prime Digit is: %d",s);
}

File: sum_of_prime_digits.py
#Python Program to Find the Sum of Prime Digits

n=int(input("Enter Your Number: "))
s=0
while (n>0):
    r=n%10
    if (n > 1):
        count=0
        for i in range(2,(r//2)+1):
            if ((r % i)==0):
                count=count+1
                break
        if (count==0):
            s=s+r
    n=n//10
print("The Sum of the Prime Digit is: ",s)